metric.go

```diff
@@ -184,6 +184,21 @@ func (m *MetricDefinition) KeyBySeries(b []byte) []byte {
 	return b
 }
 
+func (m *MetricDefinition) FullNameWithTags() string {
```

Comment:
why not call this simply `NameWithTags` ?

Reply:
I think the latter is clearer.

Reply:
can you add a comment for the function that shows what the output format is

Reply:
and a unit test is needed here too

Reply:
all done @Dieterbe